.github/workflows/carpenter.yaml

```diff
@@ -0,0 +1,22 @@
+name: Carpentry
```

Comment:
Why does this same addition appear in so many PRs?  Are the PRs layered on top of each other?

Reply:
With the monorepo being split out into separate per-plugin repos, doesn't this need to be added to each new repo?

Reply:
These seems like a bad thing to repeat if it is always the same.  Any change has to be tracked across so many different repos.  What is the value of all the separate repos having to have the same infrastructure?